packages/vendure-plugin-customer-managed-groups/src/api-extensions.ts

```diff
@@ -35,18 +36,68 @@ export const shopSchema = gql`
     firstName: String
     lastName: String
     emailAddress: String
+    addresses: [CustomerManagedGroupAddressInput]
     customerId: ID!
   }
 
+  input CustomerManagedGroupAddressInput {
+    id: ID
```

Comment:
`ID` should be mandatory, otherwise we can't update the address
```suggestion
    id: ID!
```